packetbeat/docs/fields.asciidoc

```diff
@@ -9526,7 +9537,7 @@ type: keyword
 
 --
 
-*`tls.server_hello.selected_cipher`*::
+*`tls.detailed.server_hello.selected_cipher`*::
```

Comment:
This should be stored in `tls.cipher` using the IANA names, which Packetbeat already uses, I think. (see https://ciphersuite.info/cs/TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384/). @webmat maybe we should add IANA to the ECS specification so that the field data lines up?

Reply:
@dcode Good idea. I opened this issue to track this https://github.com/elastic/ecs/issues/720

Reply:
Another field I forgot to remove. 

I don't think I follow, though. Afaik ECS docs already use an IANA name as an example (TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256).